pinot-common/src/test/java/com/linkedin/pinot/pql/parsers/Pql2CompilerTest.java

```diff
@@ -186,4 +189,20 @@ public void testQueryOptions() {
     Assert.assertEquals(brokerRequest.getQueryOptions().get("foo"), "1234");
     Assert.assertEquals(brokerRequest.getQueryOptions().get("bar"), "potato");
   }
+
+  @Test
+  public void testIdentifierQuoteCharacter() {
+    Pql2Compiler compiler = new Pql2Compiler();
+
+    TransformExpressionTree expTree = compiler.compileToExpressionTree("`a.b.c`");
+    Assert.assertEquals(expTree.getExpressionType(), TransformExpressionTree.ExpressionType.IDENTIFIER);
+    Assert.assertEquals(expTree.getValue(), "a.b.c");
+
+    BrokerRequest brokerRequest = compiler.compileToBrokerRequest(
+            "select avg(`attributes.age`) as `avg_age` from `person` group by `attributes.address_city` having avg(`attributes.age`)=20");
```

Comment:
do we support aliasing?

Reply:
Hmm, I am not certain.  I'm seeing references to aliasing in the codebase but I'm not 100% sure I follow.  Is aliasing allowing things like ```where test = :blah``` ?   And if so, are you wondering if we could also support things like ```where test = :`a.b.c` ``` ?